starter-core/src/main/java/io/micronaut/starter/feature/other/template/openApiProperties.rocker.raw

```diff
@@ -0,0 +1,6 @@
+swagger-ui.enabled=true
+redoc.enabled=false
+rapidoc.enabled=false
+rapidoc.bg-color=#14191f
+rapidoc.text-color=#aec2e0
+rapidoc.sort-endpoints-by=method
```

Comment:
I'm not sure if this is the best approach for everyone. There are other options to enable this: using system properties, configuring it in Gradle/Maven,... https://micronaut-projects.github.io/micronaut-openapi/latest/guide/index.html#openApiViews 
I don't know if all users would like this.

Also, if Micronaut Security is present the mappings probably won't work,

Reply:
I don't think it is worth complicating the PR with different configuration for the different build tools personally

Reply:
If security module is present I can add next part to config:
```
  security:
    intercept-url-map:
      - pattern: /swagger/**
        access:
          - isAnonymous()
      - pattern: /swagger-ui/**
        access:
          - isAnonymous()
```

Reply:
Be careful with that because maybe not all users want the openapi to be public. Maybe it is safest to add it but commented out and with a comment explaining it

Reply:
I have problem only with generating this part commented out because it is part of application.yaml config file. I am not sure does the Micronaut support separated config files. If it does I can create file with name "application-security.yaml" for an example but I think that the name "security" will resemble environment in this case. Separated file I could generate using the RockerTemplate instead of generating it with the snakeyaml serialization.

Reply:
personally I would keep the swagger ui publicly accessible in case the micronaut-security feature is present in context. I think that way the user can always start the app and immediately test the feature.

What would also be nice is to add annotation for the example controller so there would actually something present in the swagger-ui.

Reply:
@n0tl3ss thanks for doing this, I was looking for this feature as well. One question, you have redo/rapidoc in the properties file, but you are not generating the static resource in the application.conf file. Are you planning on adding all possible? Seems like it should at least be consistent between the config and properties file?

```yaml
  router:
    static-resources:
      swagger:
        paths: classpath:META-INF/swagger
        mapping: /swagger/**
      swagger-ui:
        paths: classpath:META-INF/swagger/views/swagger-ui
        mapping: /swagger-ui/**
      redoc:
        paths: classpath:META-INF/swagger/views/redoc
        mapping: /redoc/**
```

Reply:
@graemerocher should I add and enable redoc as well?

Reply:
@scprek With these code changes, redoc view won't be generated. In `openapi.properties` as suggested above I changed `redoc.enabled` to `false`. There are only two routes enabled, one points to all the swagger resources `/swagger` and one that makes a shortcut to swagger-ui `/swagger-ui`  which points to `/swagger/views/swagger-ui` folder. All those files/folders are in `META-INF` folder.

Reply:
Right you need to have it enabled in the open api properties and specify the routes in the app config. I was just asking if you should enable all of them and then a user can decide which to disable.

Reply:
I would leave it like this. Even if we enable redoc by default I wouldn't add the shorcut route `/redoc` to `application.yml` conifg file because when someone want to disable it and sets it to false there is potential that `/redoc` will be left as dangling route. If redoc is enabled you could access it via `/swagger/views/redoc` route.  It shouldn't be hard to add. If others agrees to add it I don't mind.

Reply:
Yeah that's a good point. I think this is fine then.